cmd/kube-proxy/app/server_others.go

```diff
@@ -25,7 +25,7 @@ import (
 	"fmt"
 	"net"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
So `proxy.ProxyProvider` → `proxy.Provider` and `EndpointsMap.UpdateEndpointsMap` → `EndpointsMap.Update` seem fine, but what is up with this? I've seen this in other PRs... something seems to do that automatically. But it's a bug, right? There's absolutely no reason to be making this change and presumably some future version of gofmt is going to want to undo it, right?

(Not specifically a question for the PR filer...)

Reply:
@danwinship This is the discussion I found around that - https://github.com/kubernetes/kubernetes/pull/76337/files#r273625182
https://github.com/kubernetes/kubernetes/issues/77045

> goimports does this now for performance reasons since the path uses a symlink and the import path isnt the real path.

Reply:
Hm, and #77045 notes "goimports aliases like v1 on core/v1 code is an annoyance that doesn't generally block PRs but it isn't any form of endorsement of the alias itself and it wouldn't hurt to discourage the practice", and also explains how to make vscode not do it (tell it to use gofmt instead of goimports). Consider yourself discouraged. I think the PR would be better without those unnecessary rewrites.

Reply:
Sure! I'll stub out those changes.

Reply:
@danwinship Done!